RELEASE.md

```diff
@@ -0,0 +1,93 @@
+# Release Checklist
+
+The following checks should be completed before publishing a new release of the
+Polkadot/Kusama/Westend runtime or client:
+
+### All Releases
+
+- [ ] Check that the new client versions have [run on the network](#burn-in) without issue for 24
```

Comment:
```suggestion
- [ ] Check that the new client versions have [run on the network](#burn-in) without issue for 12
```

24 hours seems like a lot.
I think it's reasonable to deploy the burn-in on the evening, and check the next morning that nothing went wrong.